libkbfs/tlf_journal.go

```diff
@@ -314,6 +327,26 @@ func makeTLFJournal(
 		bwDelegate:           bwDelegate,
 	}
 
+	if bws == TLFJournalBackgroundWorkPaused {
```

Comment:
Yeah thought of doing that, but didn't think it was worth changing all the callers.  Let me know if you think more commenting is needed here, or if there's another way to make it look cleaner.